ui/styles.scss

```diff
@@ -530,6 +530,29 @@ p {
     color: var(--labelSecondary) !important;
 }
 
+.custom-naming-helper-list {
+    background-color: #222222;
```

Comment:
Consider using some SCSS variables since Sketch has Light and Dark theme and such background color might not work well for both